docs/puzzles/day8.md

```diff
@@ -2,10 +2,616 @@ import Solver from "../../../../website/src/components/Solver.js"
 
 # Day 8: Seven Segment Search
 
+by [@bishabosha](https://twitter.com/bishabosha)
+
 ## Puzzle description
 
 https://adventofcode.com/2021/day/8
 
+## Solution of Part 1
+
+### Modelling the Domain
+
+First we will model our problem and parse the input into our model.
+
+#### `Segment`
+
+As there are a fixed range of possible display _segments_:
+`a`, `b`, `c`, `d`, `e`, `f`, `g`, we will model them with an enumeration.
```

Comment:
I suggest introducing an ASCIIART picture showing the 8 segments, like you did in part 2.

Reply:
I added a diagram at the start, what do you think?

Reply:
This is perfect!